apps/api_web/lib/api_web/rate_limiter.ex

```diff
@@ -35,19 +46,34 @@ defmodule ApiWeb.RateLimiter do
     @intervals_per_day
   }` |
   """
-  @spec log_request(any, String.t()) :: :ok | {:error, :rate_limited}
-  def log_request(_, "/_health" <> _), do: :ok
+  @spec log_request(any, String.t()) :: log_result
+  def log_request(_, "/_health" <> _), do: %{status: :ok}
 
   def log_request(user, _request_path) do
     max = max_requests(user)
-
-    if @limiter.rate_limited?(user.id, max) do
-      {:error, :rate_limited}
+    {key, reset_ms} = key_and_reset_time(user)
+    {rate_limited, requests} = @limiter.rate_limited?(key, max)
```

Comment:
Would it be easier to have this function return the number of requests remaining? It seems like that's what the Memcache version is returning already.